### PR TITLE
Updated README and schema

### PR DESCRIPTION
**Pull request by itay-grudev**
Fixed incorrect README after chart rename.
Updated schema and re-genarated chart README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ helm upgrade --install paradedb \
 --namespace paradedb-database \
 --create-namespace \
 --values values.yaml \
-paradedb/cluster
+paradedb/paradedb
```
Comment (itay-grudev) [inline README.md:71]: This is the actual correction to the documentation. I've replicated it in the `charts/paradedb/README.md.gotmpl` as well.

File: charts/paradedb/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# CloudNativePG Cluster
+# ParadeDB CloudNativePG Cluster
```
Comment (itay-grudev) [inline charts/paradedb/README.md:1]: This is coming from the `README.md.gotmpl`, but I think it's better this way.

File: charts/paradedb/README.md [hunk 3]
```diff
@@ -230,7 +230,7 @@ refer to  the [CloudNativePG Documentation](https://cloudnative-pg.io/documentat
 | recovery.secret.create | bool | `true` | Whether to create a secret for the backup credentials |
 | recovery.secret.name | string | `""` | Name of the backup credentials secret |
 | type | string | `"paradedb"` | Type of the CNPG database. Available types: * `paradedb` |
-| version.paradedb | string | `"0.10.0"` | The ParadeDB version, set in the publish CI workflow from the latest paradedb/paradedb GitHub tag |
+| version.paradedb | string | `"0.10.0"` | We default to v0.10.0 for testing and local development |
```
Comment (itay-grudev) [inline charts/paradedb/README.md:233]: I assume you've changed the comment in the chart `values.yaml` for `version.paradedb`.

File: charts/paradedb/values.yaml [hunk 4]
```diff
@@ -242,8 +242,10 @@ cluster:
     # -- PostgreSQL Host Based Authentication rules (lines to be appended to the pg_hba.conf file)
     pg_hba: []
       # - host all all 10.244.0.0/16 md5
+    # -- PostgreSQL User Name Maps rules (lines to be appended to the pg_ident.conf file)
     pg_ident: []
       # - mymap   /^(.*)@mydomain\.com$      \1
+    # -- Lists of shared preload libraries to add to the default ones
```
Comment (itay-grudev) [inline charts/paradedb/values.yaml:248]: These two descriptions were missing. It's a typo upstream. I've fixed it in the CNPG repo as well.

**Comment by philippemnoel** [comment]
Thank you! Pulled into #37 to run CI

**Pull request closed without merging**